integration/client/testdata/custom-1.7.toml

```diff
@@ -295,6 +295,7 @@ version = 2
     returns = "application/vnd.oci.image.layer.v1.tar+gzip"
 
 [timeouts]
+  "io.containerd.timeout.cri.defercleanup" = "1m0s"
```

Comment:
@ningmingxiao 
are we planning to cherrypick back to release 1.6 / 1.7? If not we shouldn't add it in these testdata, as the new field will only exist on config version 3.

Reply:
> if I revert it ci will failed.

can you share what's the CI error? https://github.com/containerd/containerd/pull/11380/files#r1970010230

It seems we add the filed to these 1.6/1.7 toml to work around the CI failures, but the actual feature for this field doesn't exist 1.6/1.7. So maybe we can fix the CI error.

Reply:
if testdata/default-1.6.toml and testdata/default-1.7.toml doesn't have "io.containerd.timeout.cri.defercleanup" but 
defaultContent, err := currentDefaultConfig() will have "io.containerd.timeout.cri.defercleanup" ci will failed. @djdongjin

Reply:
maybe  way is cherry pick  it into 1.6 and 1.7.

Reply:
this pr will fix https://github.com/containerd/containerd/pull/11532